tests/test_aggregation.py

```diff
@@ -16,6 +16,16 @@
 DB_URL = "sqlite+aiosqlite://"
 
 queries = [
+  (

```

Comment:
Can you add a few more tests?

Reply:
Added tests and run locally for 10 trials, all succeed.